### PR TITLE
Replace abbreviations and unified words in the Edge file

### PR DESCRIPTION
**Pull request by JoshuaAdemola**

### CODE REVIEW

File: content/edge/diagnostics-and-support-bundle/diagnostics.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ The following hardware information is available:
 |:----------------|:---|
 |CPU|Reads and gathers CPU information from "/proc/cpuinfo"
 |Memory|Reads and gathers memory information from "/proc/meminfo"
-|Detailed system summary|Data is captured using the 'lswh' command. Data includes multiple components like network, display adapter, bridge, IDE etc. This command is executed as sudo to capture all available details.
+|Detailed system summary|Data is captured using the 'lswh' command. Data includes multiple components like network, display adapter, bridge, IDE and so on. This command is executed as sudo to capture all available details.
```
Comment (BeateRixen) [inline content/edge/diagnostics-and-support-bundle/diagnostics.md:22]: I assume that there must be a comma before "and so on". Can you please advise @MWindel ?

File: content/edge/installation-bundle/accessing-platform.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ Ping the &#60;domain_name> to verify it.
 
 If the ping is successful, the DNS resolution is working properly.
 
-Using &#60;domain_name>, the Edge appliance can be connected from the host operating system (operating system which is hosting the Edge appliance). If you want to connect the Edge appliance within your LAN, which is outside of the host operating system, you need to do following:
+Using &#60;domain_name>, the Edge appliance can be connected from the host operating system (operating system which is hosting the Edge appliance). If you want to connect the Edge appliance within your LAN, which is outside of the host operating system, you must do following:
```
Comment (BeateRixen) [inline content/edge/installation-bundle/accessing-platform.md:43]: do following > do the following

File: content/edge/setting-up-edge-bundle/setting-up-vmware.md [hunk 3]
```diff
@@ -43,7 +43,7 @@ The following example illustrates the network configuration on a Windows platfor
 <img src="/images/edge/edge-vmware-06.png" name="Setting up VMware"/>
 
 7. Click **DHCP Settings** to open the **DHCP Settings** window.<br>
-8. In the fields **Starting IP address** and **Ending IP address**, change the IP range from 3 to 254, i.e. if your gateway IP is 192.168.117.2, set the IP range from 192.168.117.3 to 192.168.117.254.<br>
+8. In the fields **Starting IP address** and **Ending IP address**, change the IP range from 3 to 254, that is, if your gateway IP is 192.168.117.2, set the IP range from 192.168.117.3 to 192.168.117.254.<br>
```
Comment (BeateRixen) [inline content/edge/setting-up-edge-bundle/setting-up-vmware.md:46]: that is > that means

File: content/edge/diagnostics-and-support-bundle/diagnostics.md (hunk 1 above)
Comment (JoshuaAdemola) [reply content/edge/diagnostics-and-support-bundle/diagnostics.md:22]: Yes there is a comma.

Comment (JoshuaAdemola) [reply content/edge/diagnostics-and-support-bundle/diagnostics.md:22]: Fixed

File: content/edge/installation-bundle/accessing-platform.md (hunk 2 above)
Comment (JoshuaAdemola) [reply content/edge/installation-bundle/accessing-platform.md:43]: Fixed

File: content/edge/setting-up-edge-bundle/setting-up-vmware.md (hunk 3 above)
Comment (JoshuaAdemola) [reply content/edge/setting-up-edge-bundle/setting-up-vmware.md:46]: Fixed

**Review by MWindel — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**